Conversions/TemperatureConversion.js

```diff
@@ -0,0 +1,143 @@
+}
+
+
+const fahrenheit_to_celsius = (fahrenheit ) => {
+
+    // Wikipedia reference: https://en.wikipedia.org/wiki/Fahrenheit
+    // Wikipedia reference: https://en.wikipedia.org/wiki/Celsius
+   
+    return  Math.round(( (fahrenheit) - 32) * 5 / 9 )
+}
+
+const fahrenheit_to_kelvin = (fahrenheit ) => {
+  
+    // Wikipedia reference: https://en.wikipedia.org/wiki/Fahrenheit
+    // Wikipedia reference: https://en.wikipedia.org/wiki/Kelvin
+
+    return  Math.round((( (fahrenheit) - 32) * 5 / 9) + 273.15 )
+}
+
+const fahrenheit_to_rankine = (fahrenheit ) => {
+   
+    // Wikipedia reference: https://en.wikipedia.org/wiki/Fahrenheit
+    // Wikipedia reference: https://en.wikipedia.org/wiki/Rankine_scale
+   
+    return  Math.round( (fahrenheit) + 459.67 )
+}
+
+const kelvin_to_celsius = (kelvin ) => {
+   
+    // Wikipedia reference: https://en.wikipedia.org/wiki/Kelvin
+    // Wikipedia reference: https://en.wikipedia.org/wiki/Celsius
+    
+    return  Math.round( (kelvin) - 273.15 )
+
+}
+
+const kelvin_to_fahrenheit = (kelvin ) => {
+   
+    // Wikipedia reference: https://en.wikipedia.org/wiki/Kelvin
+    // Wikipedia reference: https://en.wikipedia.org/wiki/Fahrenheit
+   
+   
+    return  Math.round((( (kelvin) - 273.15) * 9 / 5) + 32 )
+}
+
+const kelvin_to_rankine = (kelvin ) => {
+
+    // Wikipedia reference: https://en.wikipedia.org/wiki/Kelvin
+    // Wikipedia reference: https://en.wikipedia.org/wiki/Rankine_scale
+
+    return  Math.round(( (kelvin) * 9 / 5) )
+}
+
+const rankine_to_celsius = (rankine ) => {
+   
+    // Wikipedia reference: https://en.wikipedia.org/wiki/Rankine_scale
+    // Wikipedia reference: https://en.wikipedia.org/wiki/Celsius
+  
+    return  Math.round(( (rankine) - 491.67) * 5 / 9 )
+}
+
+const rankine_to_fahrenheit = (rankine ) => {
+   
+    // Wikipedia reference: https://en.wikipedia.org/wiki/Rankine_scale
+    // Wikipedia reference: https://en.wikipedia.org/wiki/Fahrenheit
+   
+    return  Math.round( (rankine) - 459.67 )
+}
+
+
+const rankine_to_kelvin = (rankine ) => {
+   
+    // Wikipedia reference: https://en.wikipedia.org/wiki/Rankine_scale
+    // Wikipedia reference: https://en.wikipedia.org/wiki/Kelvin
+   
+    return  Math.round(( (rankine) * 5 / 9) )
+}
+
+const reaumur_to_kelvin = (reaumur ) => {
+ 
+    // Reference:- http://www.csgnetwork.com/temp2conv.html
+
+    return  Math.round(( (reaumur) * 1.25 + 273.15) )
+}
+
+const reaumur_to_fahrenheit = (reaumur ) => {
+   
+    // Reference:- http://www.csgnetwork.com/temp2conv.html
+   
+    return  Math.round(( (reaumur) * 2.25 + 32) )
+}
+
+const reaumur_to_celsius = (reaumur ) => {
+
+    // Reference:- http://www.csgnetwork.com/temp2conv.html
+
+    return  Math.round(( (reaumur) * 1.25) )
+}
+
+const reaumur_to_rankine = (reaumur ) => {
+  
+    // Reference:- http://www.csgnetwork.com/temp2conv.html
+
+    return  Math.round(( (reaumur) * 2.25 + 32 + 459.67) )
+}
+
+module.exports = {

```

Comment:
```suggestion
export {
```
This repository uses ES modules, so use the `export` statement.